### PR TITLE
Add USAConversion.netkan

### PR DESCRIPTION
**Pull request by plague006**
Breakup of #1780

### CODE REVIEW

File: NetKAN/USAConversion.netkan [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "spec_version": 1,
+    "license": "CC-BY-NC-SA-4.0",
+    "$kref": "#/ckan/kerbalstuff/962",
+    "identifier": "USAConversion",
+    "install": [
+        {
+            "file"       : "GameData/USAConversion",
```
Comment (dbent) [inline NetKAN/USAConversion.netkan]: `GameData/USAconversion` (lowercase-C) otherwise it doesn't match.

File: NetKAN/USAConversion.netkan [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "spec_version": 1,
+    "license": "CC-BY-NC-SA-4.0",
+    "$kref": "#/ckan/kerbalstuff/962",
+    "identifier": "USAConversion",
```
Comment (dbent) [inline NetKAN/USAConversion.netkan]: Also, the identifier may make more sense as `NASACareerModeTotalConversion` or something, the only reference I can find to `USAConversion` is in the install location.

**Comment by plague006** [comment]
Done

**Comment by dbent** [comment]
One of its dependencies `OMSKSpaceProducts` is currently listed as only compatible with KSP 1.0.2, might have to add an override for that. Otherwise tested manually, merging.

**Pull request merged**